pkg/cmd/run.go

```diff
@@ -1028,15 +1040,19 @@ func writeChecksumToFile(filepath string, hash hash.Hash) error {
 	return err
 }
 
-func getSpectrumOptions(platform *v1.IntegrationPlatform, cmd *cobra.Command) spectrum.Options {
+func getSpectrumOptions(platform *v1.IntegrationPlatform, cmd *cobra.Command, verbose bool) spectrum.Options {
```

Comment:
Maybe we can change it into method of `runCmdOptions` as the args now looks a bit too long.